provider/resources.go

```diff
@@ -105,13 +115,22 @@ func Provider() tfbridge.ProviderInfo {
 			// 		"tags": {Type: tfbridge.MakeType(mainPkg, "Tags")},
 			// 	},
 			// },
+			"statuscake_contact_group":      {Tok: tfbridge.MakeResource(mainPkg, mainMod, "StatuscakeContactGroup")},
```

Comment:
Currently, you have e.g this in the NodeJS SDK for resource names:

`statuscake.StatuscakeContactGroup`

But I guess it is easier to read:

`statuscake.ContactGroup`

Can you drop the `Statuscake` part from every of the resource names to remove the duplicated `statuscake` part in every resource?

Reply:
I was mapping the TF resource names but will gladly get rid of the stuttering here 👍